### PR TITLE
fix: dom handling

### PR DESCRIPTION
**Pull request by Monsterovich**

### CODE REVIEW

**Comment by Monsterovich** [comment]
@eggplants Hello!

**Comment by Monsterovich** [comment]
Interesting that if you add a delay of a second, you can keep the banners open and clicking will work. It's a strange kind of behavior.

File: deepl/deepl.py [hunk 1]
```diff
@@ -131,9 +131,11 @@ async def __translate(self, script: str) -> str:
                 # banner prevents clicking on language buttons, close the banner first
                 await page.click("button[data-testid=cookie-banner-lax-close-button]")
                 # select input / output language
-                await page.click("button[data-testid=translator-source-lang-btn]")
+                await page.click("button[data-testid=translator-source-lang-btn]", delay=1000)
```
Comment (eggplants) [inline deepl/deepl.py:134]: I dislike this because it makes each translation 1s slower than now. Please add the reason to PR description.

**Comment by Monsterovich** [comment]
@eggplants 

> I dislike this because it makes each translation 1s slower than now. Please add the reason to PR description.

I tried another option: to do click a second time. The problem is that when you do the first click something interrupts the focus and closes the list. Therefore, this problem is bypassed either via `delay` or like this.

**Comment by Monsterovich** [comment]
@eggplants Can you merge?

**Comment by Monsterovich** [comment]
@eggplants I rewrote the `click` method to `page.locator(element).dispatch_event("click")`. For some reason programmatic clicks work much better, all they need is for the element to exist in the DOM.

BTW, it works much faster. When you have time, merge this PR.

**Pull request merged**